### PR TITLE
Dashboard: Field Config - Add CFP franc currency (XPF)

### PR DESCRIPTION
**Pull request by smortex**
CFP franc has ISO 4217 currency code "XPF":
https://www.six-group.com/en/products-services/financial-information/data-standards.html

### CODE REVIEW

**Comment by smortex** [comment]
May I have a review please?

**Comment by smortex** [comment]
Hey @axelavargas, I rebased my changes on top of the `main` branch and fixed the conflicts.  Thank you!

File: devenv/dev-dashboards-without-uid/panel_tests_polystat.json [hunk 1]
```diff
@@ -448,6 +448,10 @@
             {
               "text": "Malaysian Ringgit (RM)",
               "value": "currencyMYR"
+            },
+	    {
```
Comment (axelavargas) [inline devenv/dev-dashboards-without-uid/panel_tests_polystat.json:452]: Can we fix the format (some spaces are missing)? 🙈

**Review by axelavargas — APPROVED** [review]
Hey, @smortex 👋🏾 , thank you for resolving the conflicts, I left a minor comment about formatting.

File: devenv/dev-dashboards-without-uid/panel_tests_polystat.json (hunk 1 above)
Comment (smortex) [reply devenv/dev-dashboards-without-uid/panel_tests_polystat.json]: Damned, $EDITOR put a tab when I fixed conflicts!  Commit amended, sorry for the bore.

Comment (axelavargas) [reply devenv/dev-dashboards-without-uid/panel_tests_polystat.json]: no worries 😄

**Comment by axelavargas** [comment]
Dev Notes: Levitate is showing breaking changes, but when investigating it has nothing to do with the changes in this PR.

**Pull request merged**